client/src/components/Reviews/ReviewSummary/CharacteristicBar.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const characteristicScale = {
+  Size: { min: 'Too small', mid: 'Perfect', max: 'Too large' },
+  Width: { min: 'Too narrow', mid: 'Perfect', max: 'Too wide' },
+  Comfort: { min: 'Uncomfortable', mid: 'Ok', max: 'Perfect' },
+  Quality: { min: 'Poor', mid: 'What I expected', max: 'Perfect' },
+  Length: { min: 'Too short', mid: 'Perfect', max: 'Too long' },
+  Fit: { min: 'Too tight', mid: 'Perfect', max: 'Too long' }
+};
+
+const CharacteristicBar = ({ characteristic, value }) => {
+  const percentage = Number(value) / 5 * 100;
+  return (
+    <div>
+      <div>
+        {characteristic}
+      </div>
+      <div style={{
+        display: 'flex',
+        height: '0.5em',
+        width: '100%',
+      }}>
+        {Array(3).fill(0).map((el, idx) => {
+          return (
+            <div
+              key={`bar-segment-${idx}`}
+              style={{
+                height: '90%',
+                width: '30%',
+                backgroundColor: 'gray',
+                margin: 1
+              }}
+            />
+          );
+        })}
+        <div style={{ display: 'flex', position: 'absolute', left: percentage, width: '10px' }}>
```

Comment:
I can see here what you meant when you were asking about how to customise Bootstrap. I know that it's possible to  set display/position etc with classnames but I would have done the same thing as you here since this isn't a cut-and-paste component like forms are. Just thinking out loud.